numpy/random/mtrand/mtrand.pyx

```diff
@@ -3837,20 +3837,19 @@ cdef class RandomState:
         -------
         out : ndarray or scalar
             Drawn samples from the parameterized negative binomial distribution,
-            where each sample is equal to N, the number of trials it took to
-            achieve n - 1 successes, N - (n - 1) failures, and a success on the,
-            (N + n)th trial.
+            where each sample is equal to N, the number of failures that
+            occurred before a total of n successes was reached.
```

Comment:
The description here was (probably) copied from a source with a different parametrization. I compared it with Wikipedia.

Reply:
How can you be certain what our _actual_ parameterization is? I'm not familiar with how to sample from this distribution, so the `rk_poisson(state, rk_gamma(state, n, (1-p)/p))` implementation is a little cryptic to me.